portfolio/src/main/webapp/gallery.html

```diff
@@ -9,14 +9,12 @@
 
 <body>
     <div id="content">
-     <div id="center">
-        <h1> Pictures of Jaden</h1>
-     </div>
-        <div id=img-container>
-          <img src="/Images/Jaden1.jpeg" alt="Image of Jaden" width=50% height=auto >
-          <img src="/Images/Jaden2.jpeg" alt="Jaden Graduation" width=50% height=auto>
-          <img src="/Images/Jaden3.jpeg" alt="Jaden in front of Bush Gardens christmas tree" width=50% height=auto>
-          <img src="/Images/Jaden4.jpeg" alt="Jaden eating his first meal at hampton" width=50% height=auto>
+        <h1 class="center"> Pictures of Jaden</h1>
+        <div class="img-format">
+          <img src="/Images/Jaden1.jpeg" alt="Image of Jaden" class="img">
+          <img src="/Images/Jaden2.jpeg" alt="Jaden Graduation"class="img">
+          <img src="/Images/Jaden3.jpeg" alt="Jaden in front of Bush Gardens christmas tree" class="img">
+          <img src="/Images/Jaden4.jpeg" alt="Jaden eating his first meal at hampton" class= "img">
```

Comment:
For lines like 'class= "img"', keep spacing consistent throughout the file, so this would turn into: 'class="img"'